### PR TITLE
[chassis] [masic] support multi-asic for test_route_flap

### PR DESCRIPTION
**Pull request by wenyiz2021**
### Description of PR

support test_route_flap on multi-asic, and accommodate for chassis testbeds as well
Summary:
Fixes # (issue)
https://github.com/sonic-net/sonic-mgmt/issues/7507
### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [x] 202205

#### What is the motivation for this PR?
support test_route_flap on multi-asic

#### How did you do it?
1. mark fixture announce_default_routes run for `'t0', 'm0', 'mx'` only for now to avoid error, as the API it called, `localhost.announce_routes` is flaky on t2 and t1, Cisco is working on this
2. `get_ip_route_info()` from a randomly chosen frontend asic, to support multi-asic, for single-asic it's fine too, `enum_rand_one_frontend_asic_index` will be `None`
3. for `iproute_info` getting from `show ip bgp ipv4` cli, there are 2 (or could be more) routes per route_prefix, unlike single-asic which has only one route
4. the way to get `dev_port` is the same for single-asic, but for multi-asics, we need to loop for all asics and find an external routes
5. when `check_route` to see if `dev_port` is successfully withdrawn or announced, also need to loop for all asics and gather all portchannels
6. There are min topos that don't have enough routes, skip for those topos
#### How did you verify/test it?
before:
```
2023-02-20T17:09:32.5387351Z   File "/azp/_work/58/s/tests/route/test_route_flap.py", line 188, in test_route_flap
2023-02-20T17:09:32.5389319Z     iproute_info = get_ip_route_info(duthost)
2023-02-20T17:09:32.5391287Z   File "/azp/_work/58/s/tests/route/test_route_flap.py", line 145, in get_ip_route_info
2023-02-20T17:09:32.5394360Z     output = json.loads(duthost.shell('vtysh -c "show ip bgp ipv4 json"', verbose=False)['stdout'])
2023-02-20T17:09:32.5396763Z   File "/azp/_work/58/s/tests/common/devices/multi_asic.py", line 108, in _run_on_asics
2023-02-20T17:09:32.5399119Z     return getattr(self.sonichost, self.multi_asic_attr)(*module_args, **complex_args)
2023-02-20T17:09:32.5401304Z   File "/azp/_work/58/s/tests/common/devices/base.py", line 89, in _run
2023-02-20T17:09:32.5403414Z     raise RunAnsibleModuleFail("run module {} failed".format(self.module_name), res)
2023-02-20T17:09:32.5405519Z RunAnsibleModuleFail: run module shell failed, Ansible Results =>
2023-02-20T17:09:32.5411601Z {"changed": true, "cmd": "vtysh -c \"show ip bgp ipv4 json\"", "delta": "0:00:00.010329", "end": "2023-02-20 17:09:32.267200", "failed": true, "msg": "non-zero return code", "rc": 1, "start": "2023-02-20 17:09:32.256871", "stderr": "Usage: /usr/bin/vtysh -n [0 to 1] [OPTION]... ", "stderr_lines": ["Usage: /usr/bin/vtysh -n [0 to 1] [OPTION]... "], "stdout": "", "stdout_lines": []}
```

after:
```
route/test_route_flap.py::test_route_flap[2] 
------------------------------------------------------------------------------------------------------------------- live log collection -------------------------------------------------------------------------------------------------------------------
21:48:29 issue.is_active                          L0085 ERROR  | Get details for https://github.com/sonic-net/sonic-mgmt/issues/6668 failed with: ConnectionError(MaxRetryError("HTTPSConnectionPool(host='api.github.com', port=443): Max retries exceeded with url: /repos/sonic-net/sonic-mgmt/issues/6668 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fe0c02c5210>: Failed to establish a new connection: [Errno 110] Connection timed out',))",),)
PASSED                                                                                                                                                                                                 [100%]
```

on single-asic dut, after change:
```
wenyizhang@e8e9d01adbfb:/var/src/sonic-mgmt-int/tests$ sudo ./run_tests.sh -i '../ansible/str2,../ansible/veos' -n vms20-t0-7050cx3-2 -u -e "--disable_loganalyzer --skip_sanity" -c route/test_route_flap.py
=== Running tests in groups ===
Running: pytest route/test_route_flap.py --inventory ../ansible/str2,../ansible/veos --host-pattern str2-7050cx3-acs-02 --testbed vms20-t0-7050cx3-2 --testbed_file /var/src/sonic-mgmt-int/ansible/testbed.yaml --log-cli-level warning --log-file-level debug --kube_master unset --showlocals --assert plain --show-capture no -rav --allow_recover --ignore=ptftests --ignore=acstests --ignore=saitests --ignore=scripts --ignore=k8s --ignore=sai_qualify --junit-xml=logs/tr.xml --log-file=logs/test.log --disable_loganalyzer --skip_sanity
/usr/local/lib/python2.7/dist-packages/ansible/parsing/vault/__init__.py:44: CryptographyDeprecationWarning: Python 2 is no longer supported by the Python core team. Support for it is now deprecated in cryptography, and will be removed in the next release.
  from cryptography.exceptions import InvalidSignature
================================================================================================================== test session starts ===================================================================================================================                                                                                                                                                                            

route/test_route_flap.py::test_route_flap[None] 
------------------------------------------------------------------------------------------------------------------ live log collection -------------------------------------------------------------------------------------------------------------------
21:22:55 issue.is_active                          L0085 ERROR  | Get details for https://github.com/sonic-net/sonic-mgmt/issues/6668 failed with: ConnectionError(MaxRetryError("HTTPSConnectionPool(host='api.github.com', port=443): Max retries exceeded with url: /repos/sonic-net/sonic-mgmt/issues/6668 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f6b23754350>: Failed to establish a new connection: [Errno 110] Connection timed out',))",),)
PASSED                                                                                                                                                                                             [100%]
```

on min topo:
```
SKIPPED [1] /var/src/private/sonic-mgmt-int/tests/route/test_route_flap.py:262: Skip this test for current topo.                    At least 1 multipath route coming from ebgp is needed!
========================= 1 skipped in 264.99 seconds ==========================
```

### CODE REVIEW

**Comment by wangxin** [comment]
The [pre-commit](http://pre-commit.com/) check detected issues in the files touched by this pull request.
The detected issues may be old or new. For new issues, please try to fix them.

For old issues, it is not mandatory to fix them because they were not caused by this change. It is unfair to blame
author of this pull request. But if you can take extra effort to fix the old issues as well, that would be great!

Detailed pre-commit check results:
<samp>trim trailing whitespace.................................................Failed<br/>- hook id: trailing-whitespace<br/>- exit code: 1<br/>- files were modified by this hook<br/><br/>Fixing tests/route/test_route_flap.py<br/><br/>fix end of files.........................................................Passed<br/>check yaml...........................................(no files to check)Skipped<br/>check for added large files..............................................Passed<br/>check python ast.........................................................Passed<br/>flake8...................................................................Failed<br/>- hook id: flake8<br/>- exit code: 1<br/><br/>tests/route/test_route_flap.py:108:84: F821 undefined name 'asic_idx'<br/>tests/route/test_route_flap.py:108:121: E501 line too long (126 > 120 characters)<br/>tests/route/test_route_flap.py:119:47: E251 unexpected spaces around keyword / parameter equals<br/>tests/route/test_route_flap.py:119:49: E251 unexpected spaces around keyword / parameter equals<br/>tests/route/test_route_flap.py:119:66: E251 unexpected spaces around keyword / parameter equals<br/>...<br/>[truncated extra lines, please run pre-commit locally to view full check results]<br/></samp>

To run the pre-commit checks locally, you can follow below steps:
1. Ensure that default python is python3. In sonic-mgmt docker container, default python is python2. You can run
   the check by activating the python3 virtual environment in sonic-mgmt docker container or outside of sonic-mgmt
   docker container.
2. Ensure that the `pre-commit` package is installed:
```
sudo pip install pre-commit
```
3. Go to repository root folder
4. Install the pre-commit hooks:
```
pre-commit install
```
5. Use pre-commit to check staged file:
```
pre-commit
```
6. Alternatively, you can check committed files using:
```
pre-commit run --from-ref <commit_id> --to-ref <commit_id>
```

**Comment by wangxin** [comment]
The [pre-commit](http://pre-commit.com/) check detected issues in the files touched by this pull request.
The detected issues may be old or new. For new issues, please try to fix them.

For old issues, it is not mandatory to fix them because they were not caused by this change. It is unfair to blame
author of this pull request. But if you can take extra effort to fix the old issues as well, that would be great!

Detailed pre-commit check results:
<samp>trim trailing whitespace.................................................Failed<br/>- hook id: trailing-whitespace<br/>- exit code: 1<br/>- files were modified by this hook<br/><br/>Fixing tests/route/test_route_flap.py<br/><br/>fix end of files.........................................................Passed<br/>check yaml...........................................(no files to check)Skipped<br/>check for added large files..............................................Passed<br/>check python ast.........................................................Passed<br/>flake8...................................................................Failed<br/>- hook id: flake8<br/>- exit code: 1<br/><br/>tests/route/test_route_flap.py:120:47: E251 unexpected spaces around keyword / parameter equals<br/>tests/route/test_route_flap.py:120:49: E251 unexpected spaces around keyword / parameter equals<br/>tests/route/test_route_flap.py:120:66: E251 unexpected spaces around keyword / parameter equals<br/>tests/route/test_route_flap.py:120:68: E251 unexpected spaces around keyword / parameter equals<br/>tests/route/test_route_flap.py:120:84: E251 unexpected spaces around keyword / parameter equals<br/>...<br/>[truncated extra lines, please run pre-commit locally to view full check results]<br/></samp>

To run the pre-commit checks locally, you can follow below steps:
1. Ensure that default python is python3. In sonic-mgmt docker container, default python is python2. You can run
   the check by activating the python3 virtual environment in sonic-mgmt docker container or outside of sonic-mgmt
   docker container.
2. Ensure that the `pre-commit` package is installed:
```
sudo pip install pre-commit
```
3. Go to repository root folder
4. Install the pre-commit hooks:
```
pre-commit install
```
5. Use pre-commit to check staged file:
```
pre-commit
```
6. Alternatively, you can check committed files using:
```
pre-commit run --from-ref <commit_id> --to-ref <commit_id>
```

**Comment by wangxin** [comment]
The [pre-commit](http://pre-commit.com/) check detected issues in the files touched by this pull request.
The detected issues may be old or new. For new issues, please try to fix them.

For old issues, it is not mandatory to fix them because they were not caused by this change. It is unfair to blame
author of this pull request. But if you can take extra effort to fix the old issues as well, that would be great!

Detailed pre-commit check results:
<samp>trim trailing whitespace.................................................Failed<br/>- hook id: trailing-whitespace<br/>- exit code: 1<br/>- files were modified by this hook<br/><br/>Fixing tests/route/test_route_flap.py<br/><br/>fix end of files.........................................................Passed<br/>check yaml...........................................(no files to check)Skipped<br/>check for added large files..............................................Passed<br/>check python ast.........................................................Passed<br/>flake8...................................................................Failed<br/>- hook id: flake8<br/>- exit code: 1<br/><br/>tests/route/test_route_flap.py:109:5: F841 local variable 'ns_cmd' is assigned to but never used<br/>tests/route/test_route_flap.py:121:47: E251 unexpected spaces around keyword / parameter equals<br/>tests/route/test_route_flap.py:121:49: E251 unexpected spaces around keyword / parameter equals<br/>tests/route/test_route_flap.py:121:66: E251 unexpected spaces around keyword / parameter equals<br/>tests/route/test_route_flap.py:121:68: E251 unexpected spaces around keyword / parameter equals<br/>...<br/>[truncated extra lines, please run pre-commit locally to view full check results]<br/></samp>

To run the pre-commit checks locally, you can follow below steps:
1. Ensure that default python is python3. In sonic-mgmt docker container, default python is python2. You can run
   the check by activating the python3 virtual environment in sonic-mgmt docker container or outside of sonic-mgmt
   docker container.
2. Ensure that the `pre-commit` package is installed:
```
sudo pip install pre-commit
```
3. Go to repository root folder
4. Install the pre-commit hooks:
```
pre-commit install
```
5. Use pre-commit to check staged file:
```
pre-commit
```
6. Alternatively, you can check committed files using:
```
pre-commit run --from-ref <commit_id> --to-ref <commit_id>
```

**Comment by wangxin** [comment]
The [pre-commit](http://pre-commit.com/) check detected issues in the files touched by this pull request.
The detected issues may be old or new. For new issues, please try to fix them.

For old issues, it is not mandatory to fix them because they were not caused by this change. It is unfair to blame
author of this pull request. But if you can take extra effort to fix the old issues as well, that would be great!

Detailed pre-commit check results:
<samp>trim trailing whitespace.................................................Failed<br/>- hook id: trailing-whitespace<br/>- exit code: 1<br/>- files were modified by this hook<br/><br/>Fixing tests/route/test_route_flap.py<br/><br/>fix end of files.........................................................Passed<br/>check yaml...........................................(no files to check)Skipped<br/>check for added large files..............................................Passed<br/>check python ast.........................................................Passed<br/>flake8...................................................................Failed<br/>- hook id: flake8<br/>- exit code: 1<br/><br/>tests/route/test_route_flap.py:121:47: E251 unexpected spaces around keyword / parameter equals<br/>tests/route/test_route_flap.py:121:49: E251 unexpected spaces around keyword / parameter equals<br/>tests/route/test_route_flap.py:121:66: E251 unexpected spaces around keyword / parameter equals<br/>tests/route/test_route_flap.py:121:68: E251 unexpected spaces around keyword / parameter equals<br/>tests/route/test_route_flap.py:121:84: E251 unexpected spaces around keyword / parameter equals<br/>...<br/>[truncated extra lines, please run pre-commit locally to view full check results]<br/></samp>

To run the pre-commit checks locally, you can follow below steps:
1. Ensure that default python is python3. In sonic-mgmt docker container, default python is python2. You can run
   the check by activating the python3 virtual environment in sonic-mgmt docker container or outside of sonic-mgmt
   docker container.
2. Ensure that the `pre-commit` package is installed:
```
sudo pip install pre-commit
```
3. Go to repository root folder
4. Install the pre-commit hooks:
```
pre-commit install
```
5. Use pre-commit to check staged file:
```
pre-commit
```
6. Alternatively, you can check committed files using:
```
pre-commit run --from-ref <commit_id> --to-ref <commit_id>
```

**Comment by arlakshm** [comment]
@wenyiz2021, can you update the description with more details on what changes are being made in this PR and why we need them?

**Comment by wenyiz2021** [comment]
> @wenyiz2021, can you update the description with more details on what changes are being made in this PR and why we need them?

sure, I'll add once I thouroughly tested out

**Comment by wangxin** [comment]
The [pre-commit](http://pre-commit.com/) check detected issues in the files touched by this pull request.
The detected issues may be old or new. For new issues, please try to fix them.

For old issues, it is not mandatory to fix them because they were not caused by this change. It is unfair to blame
author of this pull request. But if you can take extra effort to fix the old issues as well, that would be great!

Detailed pre-commit check results:
<samp>trim trailing whitespace.................................................Failed<br/>- hook id: trailing-whitespace<br/>- exit code: 1<br/>- files were modified by this hook<br/><br/>Fixing tests/route/test_route_flap.py<br/><br/>fix end of files.........................................................Passed<br/>check yaml...........................................(no files to check)Skipped<br/>check for added large files..............................................Passed<br/>check python ast.........................................................Passed<br/>flake8...................................................................Failed<br/>- hook id: flake8<br/>- exit code: 1<br/><br/>tests/route/test_route_flap.py:40:1: E302 expected 2 blank lines, found 1<br/>tests/route/test_route_flap.py:113:121: E501 line too long (134 > 120 characters)<br/>tests/route/test_route_flap.py:118:121: E501 line too long (121 > 120 characters)<br/>tests/route/test_route_flap.py:128:47: E251 unexpected spaces around keyword / parameter equals<br/>tests/route/test_route_flap.py:128:49: E251 unexpected spaces around keyword / parameter equals<br/>...<br/>[truncated extra lines, please run pre-commit locally to view full check results]<br/></samp>

To run the pre-commit checks locally, you can follow below steps:
1. Ensure that default python is python3. In sonic-mgmt docker container, default python is python2. You can run
   the check by activating the python3 virtual environment in sonic-mgmt docker container or outside of sonic-mgmt
   docker container.
2. Ensure that the `pre-commit` package is installed:
```
sudo pip install pre-commit
```
3. Go to repository root folder
4. Install the pre-commit hooks:
```
pre-commit install
```
5. Use pre-commit to check staged file:
```
pre-commit
```
6. Alternatively, you can check committed files using:
```
pre-commit run --from-ref <commit_id> --to-ref <commit_id>
```

**Comment by wangxin** [comment]
The [pre-commit](http://pre-commit.com/) check detected issues in the files touched by this pull request.
The detected issues may be old or new. For new issues, please try to fix them.

For old issues, it is not mandatory to fix them because they were not caused by this change. It is unfair to blame
author of this pull request. But if you can take extra effort to fix the old issues as well, that would be great!

Detailed pre-commit check results:
<samp>trim trailing whitespace.................................................Failed<br/>- hook id: trailing-whitespace<br/>- exit code: 1<br/>- files were modified by this hook<br/><br/>Fixing tests/route/test_route_flap.py<br/><br/>fix end of files.........................................................Passed<br/>check yaml...........................................(no files to check)Skipped<br/>check for added large files..............................................Passed<br/>check python ast.........................................................Passed<br/>flake8...................................................................Failed<br/>- hook id: flake8<br/>- exit code: 1<br/><br/>tests/route/test_route_flap.py:40:1: E302 expected 2 blank lines, found 1<br/>tests/route/test_route_flap.py:113:121: E501 line too long (134 > 120 characters)<br/>tests/route/test_route_flap.py:118:121: E501 line too long (121 > 120 characters)<br/>tests/route/test_route_flap.py:128:47: E251 unexpected spaces around keyword / parameter equals<br/>tests/route/test_route_flap.py:128:49: E251 unexpected spaces around keyword / parameter equals<br/>...<br/>[truncated extra lines, please run pre-commit locally to view full check results]<br/></samp>

To run the pre-commit checks locally, you can follow below steps:
1. Ensure that default python is python3. In sonic-mgmt docker container, default python is python2. You can run
   the check by activating the python3 virtual environment in sonic-mgmt docker container or outside of sonic-mgmt
   docker container.
2. Ensure that the `pre-commit` package is installed:
```
sudo pip install pre-commit
```
3. Go to repository root folder
4. Install the pre-commit hooks:
```
pre-commit install
```
5. Use pre-commit to check staged file:
```
pre-commit
```
6. Alternatively, you can check committed files using:
```
pre-commit run --from-ref <commit_id> --to-ref <commit_id>
```

**Comment by wangxin** [comment]
The [pre-commit](http://pre-commit.com/) check detected issues in the files touched by this pull request.
The detected issues may be old or new. For new issues, please try to fix them.

For old issues, it is not mandatory to fix them because they were not caused by this change. It is unfair to blame
author of this pull request. But if you can take extra effort to fix the old issues as well, that would be great!

Detailed pre-commit check results:
<samp>trim trailing whitespace.................................................Failed<br/>- hook id: trailing-whitespace<br/>- exit code: 1<br/>- files were modified by this hook<br/><br/>Fixing tests/route/test_route_flap.py<br/><br/>fix end of files.........................................................Passed<br/>check yaml...........................................(no files to check)Skipped<br/>check for added large files..............................................Passed<br/>check python ast.........................................................Passed<br/>flake8...................................................................Failed<br/>- hook id: flake8<br/>- exit code: 1<br/><br/>tests/route/test_route_flap.py:114:121: E501 line too long (134 > 120 characters)<br/>tests/route/test_route_flap.py:119:121: E501 line too long (121 > 120 characters)<br/>tests/route/test_route_flap.py:129:47: E251 unexpected spaces around keyword / parameter equals<br/>tests/route/test_route_flap.py:129:49: E251 unexpected spaces around keyword / parameter equals<br/>tests/route/test_route_flap.py:129:66: E251 unexpected spaces around keyword / parameter equals<br/>...<br/>[truncated extra lines, please run pre-commit locally to view full check results]<br/></samp>

To run the pre-commit checks locally, you can follow below steps:
1. Ensure that default python is python3. In sonic-mgmt docker container, default python is python2. You can run
   the check by activating the python3 virtual environment in sonic-mgmt docker container or outside of sonic-mgmt
   docker container.
2. Ensure that the `pre-commit` package is installed:
```
sudo pip install pre-commit
```
3. Go to repository root folder
4. Install the pre-commit hooks:
```
pre-commit install
```
5. Use pre-commit to check staged file:
```
pre-commit
```
6. Alternatively, you can check committed files using:
```
pre-commit run --from-ref <commit_id> --to-ref <commit_id>
```

**Comment by wangxin** [comment]
The [pre-commit](http://pre-commit.com/) check detected issues in the files touched by this pull request.
The detected issues may be old or new. For new issues, please try to fix them.

For old issues, it is not mandatory to fix them because they were not caused by this change. It is unfair to blame
author of this pull request. But if you can take extra effort to fix the old issues as well, that would be great!

Detailed pre-commit check results:
<samp>trim trailing whitespace.................................................Failed<br/>- hook id: trailing-whitespace<br/>- exit code: 1<br/>- files were modified by this hook<br/><br/>Fixing tests/route/test_route_flap.py<br/><br/>fix end of files.........................................................Passed<br/>check yaml...........................................(no files to check)Skipped<br/>check for added large files..............................................Passed<br/>check python ast.........................................................Failed<br/>- hook id: check-ast<br/>- exit code: 1<br/><br/>tests/route/test_route_flap.py: failed parsing with CPython 3.10.6:<br/><br/>Traceback (most recent call last):<br/>File "/home/vsts/.cache/pre-commit/repoaf3k7y0m/py_env-python3/lib/python3.10/site-packages/pre_commit_hooks/check_ast.py", line 21, in main<br/>ast.parse(f.read(), filename=filename)<br/>File "/usr/lib/python3.10/ast.py", line 50, in parse<br/>...<br/>[truncated extra lines, please run pre-commit locally to view full check results]<br/></samp>

To run the pre-commit checks locally, you can follow below steps:
1. Ensure that default python is python3. In sonic-mgmt docker container, default python is python2. You can run
   the check by activating the python3 virtual environment in sonic-mgmt docker container or outside of sonic-mgmt
   docker container.
2. Ensure that the `pre-commit` package is installed:
```
sudo pip install pre-commit
```
3. Go to repository root folder
4. Install the pre-commit hooks:
```
pre-commit install
```
5. Use pre-commit to check staged file:
```
pre-commit
```
6. Alternatively, you can check committed files using:
```
pre-commit run --from-ref <commit_id> --to-ref <commit_id>
```

**Comment by wenyiz2021** [comment]
> @wenyiz2021, can you update the description with more details on what changes are being made in this PR and why we need them?

Hi @arlakshm , I have thouroughly tested out and updated details of changes in PR description

File: tests/route/test_route_flap.py
Comment (wenyiz2021) [reply tests/route/test_route_flap.py:56]: hi @arlakshm , this is a fixture at test teardown, which calls ` localhost.announce_routes` that is not designed for other topos I believe, the only other caller of this `announce_routes` is only marking for 't0', 'm0', 'mx'

https://github.com/sonic-net/sonic-mgmt/blob/2d3d85c259fc8806acc74bfc4e2564965be32d04/tests/stress/test_stress_routes.py#L18

**Comment by abdosi** [comment]
@vperumal can you please check on this PR.

File: tests/route/test_route_flap.py
Comment (veronica-arista) [inline tests/route/test_route_flap.py:120]: This looks like an indentation error.

Comment (wenyiz2021) [reply tests/route/test_route_flap.py]: thanks, fixed

File: tests/route/test_route_flap.py
Comment (kartik-arista) [inline tests/route/test_route_flap.py:229]: Is the `and 'IB'` check ncessary ? Would it not be enough to just check for membership in internal_peers? 

The reason I ask is that if for any reason the naming scheme changes down the road, then this test will unnecessarily break. It is better not to depend on naming format.

Comment (kartik-arista) [reply tests/route/test_route_flap.py:229]: Okay thats fine with me.  Feel free to merge this if there are no otehr issues.

Comment (wenyiz2021) [reply tests/route/test_route_flap.py:229]: this testcase seems flaky on arista testbed, depending on which ports/routes it takes

I'll check the issue described in https://github.com/sonic-net/sonic-mgmt/issues/7507 before merging

Comment (wenyiz2021) [reply tests/route/test_route_flap.py]: this comment is addressed

File: tests/route/test_route_flap.py
Comment (veronica-arista) [inline tests/route/test_route_flap.py:166]: Should this function be changed as well, given that the latest commit removes get_internal_bgp_peers_for_chassis_per_asic() ?

**Comment by wenyiz2021** [comment]
As of today May 5th:
Arista chassis consistently passing with 3 runs
Also run on another single-asic dut, multi-asic dut, it's passing

**Comment by wenyiz2021** [comment]
@veronica-arista @kartik-arista @arlakshm kindly request review from you

**Review by abdosi — APPROVED** [review]
LGTM

**Comment by wangxin** [comment]
@wenyiz2021 The 202012 branch does not have this test_rotue_flap.py script at all. I am removing label "Request for 202012 branch".

**Pull request merged**
